src/lualib/ArrayPush.ts

```diff
@@ -1,6 +1,8 @@
 function __TS__ArrayPush<T>(arr: T[], ...items: T[]): number {
-    for (const item of items) {
-        arr[arr.length] = item;
+    /* tslint:disable */
```

Comment:
We shouldn't be disabling linting like this. Instead we should just transpile `for ... of` using a numeric loop to solve it at that level.

Reply:
Not really in the scope of this PR.
We could also just disable the for...orf rule in our tslint config, I'm not a big fan of that rule anyway.